src/material-experimental/mdc-table/table.scss

```diff
@@ -15,3 +19,69 @@
 .mdc-data-table__table {
   table-layout: auto;
 }
+
+/**
+ * Flex-based table structure
+ */
+mat-table {
```

Comment:
Rather than duplicating these styles between the two implementations, could we exact them into a private mixin in `material/table` that both places include?